task8/src/main/webapp/resources/js/AdCollection.js

```diff
@@ -0,0 +1,388 @@
+let adList = [
```

Comment:
You understand that from now on, the son storage is not needed, because data will come from the backend, right?) Technically you were expected to start consuming the JSON produced by the servlet